developer-docs-site/docs/reference/move.md

```diff
@@ -0,0 +1,110 @@
+
+The Aptos framework ships with many useful libraries:
+* A [Token standard](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/Token.move) that makes it possible to create NFTs and other rich tokens without publishing a smart contract
+* A [Coin standard](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/Coin.move) that makes it possible to create typesafe Coins by publishing a trivial module
+* An [iterable Table](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/IterableTable.move) that allows for traversing all the keys within a the Table
+* A staking and delegation framework
+* A [`type_of`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/TypeInfo.move) service to identify at run-time the address, module, and struct name of a given type
+* Multi-signer framework that allows multiple `signer` entities
+* A [timestamp service](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/Timestamp.move) that provides a monotonically increasing clock that maps close to the actual current unixtime
+
+With much more coming soon...
+
+## Key Concepts in Move
+
+* Data should be stored within the account that owns it not the account that published the module.
+* Data flow should have miimal constraints with an emphasis on ecosystem usability
+* Prefer static typesafety over run-time safety via generics
+* A `signer` should be required to restrict access to adding or removing assets to an account unless it is explicitly clear
+
+### Data ownership
+
+Data should be stored within the account that owns it not the account that published the module.
+
+In Solidity, data is stored within the namespace of the account that created the contract. Typically this is represented by a map of an address to a value or of an object id to the address of the owner.
+
+In Solana, data is stored within a distinct account associated with a contract.
+
+In Move, data can be stored within the module owner's account, but that creates the issue of ownership ambiguity and implies that two issues:
+1. It makes ownership ambiguous as the asset has no resource associated with the owner
+2. The module creator takes responsibility for the lifetime of that resources, e.g., rent, reclamation, etc
+
+On the first point, by placing assets within trusted resources within an account, the owner can ensure that even a maliciously programmed module will be unable to modify those assets. In Move, we can program a standard orderbook structure and interface that would let applications built on top be unable to gain backdoor access to an account or its orderbook entries.
+
+### Data flow
+
+Data flow should have miimal constraints with an emphasis on ecosystem usability
+
```

Comment:
It's kinda weird how the first one you compare to other chains then these you don't.

Reply:
this was the one that is most obvious in samples of code and it wasnt' called out in earlier sections.